src/trips-repo.js

```diff
@@ -23,6 +32,20 @@ class TripsRepository {
     return this.userTrips
   }
 
+  sortUserTrips(date){
```

Comment:
This bit of sorting is interesting. I think it'll ultimately make for a very user-friendly DOM.